### PR TITLE
feat: improvements on glossary compile

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

feat: some improvements

1. not all browsers support all functions in aspect of wasm, even in some latest versions of major browsers
2. link improvements

### CODE REVIEW

File: files/en-us/glossary/compile/index.md [hunk 1]
```diff
@@ -20,6 +20,5 @@ Compilers may also translate among higher-level languages — for example, from
 
 ## See also
 
-- [Compiling from C/C++ to WebAssembly](/en-US/docs/WebAssembly/C_to_Wasm)
-- [Compiling from Rust to WebAssembly](/en-US/docs/WebAssembly/Rust_to_Wasm)
```
Comment (PassionPenguin) [inline files/en-us/glossary/compile/index.md:24]: replacing with a higher level doc should be better?

File: files/en-us/glossary/compile/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ page-type: glossary-definition
...
+Typically, a compiler transforms code written in a higher-level language such as [C++](https://en.wikipedia.org/wiki/C++) or [Rust](<https://en.wikipedia.org/wiki/Rust_(programming_language)>) or [Java](<https://en.wikipedia.org/wiki/Java_(programming_language)>) into executable (runnable) code — so-called **binary code** or **machine code**. [WebAssembly](/en-US/docs/WebAssembly), for example, is a form of executable binary code that [can be compiled from code written in C++, Rust, C#, Go, Swift, and several other languages](https://webassembly.org/getting-started/developers-guide/) and that can then be run on any web page, with most features supported in modern browsers (see [browser compatibility table](/en-US/docs/WebAssembly#browser_compatibility)).
```
Comment (chrisdavidmills) [inline files/en-us/glossary/compile/index.md:11]:
```suggestion
Typically, a compiler transforms code written in a higher-level language such as [C++](https://en.wikipedia.org/wiki/C++), [Rust](<https://en.wikipedia.org/wiki/Rust_(programming_language)>), or [Java](<https://en.wikipedia.org/wiki/Java_(programming_language)>) into executable (runnable) code — so-called **binary code** or **machine code**. [WebAssembly](/en-US/docs/WebAssembly), for example, is a form of executable binary code that [can be compiled from code written in C++, Rust, C#, Go, Swift, and several other languages](https://webassembly.org/getting-started/developers-guide/) and run on any web page, with most features supported in modern browsers (see [browser compatibility table](/en-US/docs/WebAssembly#browser_compatibility)).
```

File: files/en-us/glossary/compile/index.md (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/glossary/compile/index.md:24]: Yup, agreed!

File: files/en-us/glossary/compile/index.md [hunk 3]
```diff
@@ -20,6 +20,5 @@ Compilers may also translate among higher-level languages — for example, from
 
 ## See also
 
-- [Compiling from C/C++ to WebAssembly](/en-US/docs/WebAssembly/C_to_Wasm)
-- [Compiling from Rust to WebAssembly](/en-US/docs/WebAssembly/Rust_to_Wasm)
-- Wikipedia: [Compiler](https://en.wikipedia.org/wiki/Compiler)
+- [WebAssembly](/en-US/docs/WebAssembly)
+- [Compiler](https://en.wikipedia.org/wiki/Compiler) on Wikipedia
```
Comment (chrisdavidmills) [inline files/en-us/glossary/compile/index.md:24]: nit: maybe swap the order of the two links, as the "Compiler" link is more directly relevant to the subject matter?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@PassionPenguin this is looking pretty good, nice work! I just had a couple of small suggestions for you to look at.

File: files/en-us/glossary/compile/index.md (hunk 3 above)
Comment (PassionPenguin) [reply files/en-us/glossary/compile/index.md:24]: absolutely yes! Will have a patch right away!

**Review by chrisdavidmills — APPROVED** [review]
@PassionPenguin perfecto! Let's get this one merged.

Thanks again.

**Pull request merged**